docker/beta/uwsgi/uwsgi.service

```diff
@@ -34,5 +34,11 @@ then
     exec uwsgi /etc/uwsgi/uwsgi-api-compat.ini
 fi
 
+if [ "${CONTAINER_NAME}" = "listenbrainz-follow-dispatch-beta" ]
+then
+    cd /code/listenbrainz
+    exec python3 manage.py run_follow_server -h 0.0.0.0 -p 8081
```

Comment:
I don't think this is the ideal way to run the server in production, uwsgi?

Reply:
The whole situation of how to run in production is entirely unclear right now. For now, we can let this go, but for production this needs careful consideration. I'm OK to merge as is.